sycl/include/CL/sycl/intel/builtins.hpp

```diff
@@ -0,0 +1,69 @@
+//==------ builtins.hpp - Non-standard SYCL built-in functions -------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+
+#ifdef __SYCL_DEVICE_ONLY__
+#define CONSTANT_AS __attribute__((opencl_constant))
+// Note: __format string is declared in constant address space to be compatible
+// with OpenCL C
+extern int __spirv_ocl_printf(const CONSTANT_AS char *__format, ...)
+    __attribute__((format(printf, 1, 2)));
```

Comment:
The `format` attribute is not aware about the fact that a format
string is interpreted according to the OpenCL C spec. It is going
emit a false warning when types actually mismatch between host and
OpenCL.

Reply:
Removed the attribute